ddtrace/profile/collector/__init__.py

```diff
@@ -0,0 +1,137 @@
+# -*- encoding: utf-8 -*-
+import abc
+
+from ddtrace.vendor import six
+
+from ddtrace.profile import _attr
+from ddtrace.profile import _periodic
+from ddtrace.vendor import attr
+
+
+# This ought to use `enum.Enum`, but since it's not available in Python 2, we just use a dumb class.
```

Comment:
https://pypi.org/project/enum34/

I think we are already using this for something else.